### PR TITLE
Mindslave and Greytide implant antag hud

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

Mindslave and Greytide implant antag hud
mindslave
#### Changelog

:cl:  
rscadd: Mindslave and Greytide implant antag hud
/:cl:

### CODE REVIEW

**Comment by jcatjmeow** [comment]
https://forums.yogstation.net/index.php?threads/a-guide-to-testing-your-prs-before-you-pr-them.22028/
SCREAM

File: code/datums/hud.dm [hunk 1]
```diff
@@ -35,6 +35,7 @@ GLOBAL_LIST_INIT(huds, list(
 	ANTAG_HUD_DARKSPAWN = new/datum/atom_hud/antag(), //Yogs
 	ANTAG_HUD_CAPITALIST = new/datum/atom_hud/antag(),
 	ANTAG_HUD_COMMUNIST = new/datum/atom_hud/antag()
```
Comment (jcatjmeow) [inline code/datums/hud.dm]:
```suggestion
	ANTAG_HUD_COMMUNIST = new/datum/atom_hud/antag(),
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

**Review by jcatjmeow — APPROVED** [review]
Gave it a once over and it looks good, if it's tested then woot

**Comment by MegaEmpirical** [comment]
Fucking finally, good work.

**Review by SomeguyManperson — DISMISSED** [review]
need to actually call the new procs probably in apply_innate_effects for the antag or they won't do anything

**Comment by SomeguyManperson** [comment]
wrong button aaaa

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**